source/support.tex

```diff
@@ -580,6 +580,7 @@
   // also in any C++ library header from \tref{headers.cpp} or any C++ header for C library facilities from \tref{headers.cpp.c}
 #define @\defnlibxname{cpp_lib_constexpr_dynamic_alloc}@           201907L // also in \libheader{memory}
 #define @\defnlibxname{cpp_lib_constexpr_invoke}@                  201907L // also in \libheader{functional}
+#define @\defnlibxname{cpp_lib_constexpr_numeric}@                 201911L // also in \libheader{numeric}
```

Comment:
The paper says "cpp_lib_constexpr_numeric_algorithms".

Reply:
Both [the revision of `d1645r1.html` we approved in LWG](http://wiki.edg.com/pub/Wg21belfast/LibraryWorkingGroup/d1645r1.html) and the [P1645R1 attached to the StrawPolls page](http://wiki.edg.com/pub/Wg21belfast/StrawPolls/P1645R1__constexpr_for__numeric__algorithms.html) use `__cpp_lib_constexpr_numeric`.

Reply:
@CaseyCarter , you're right. I have no idea where I looked to find the other spelling.